### ISSUE TITLE
FeatureRequest: work with default ssh config

### ISSUE THREAD

**Issue by 13werwolf13** [ratsky-oss/sshabu#1]
when working in large teams, often `~/.ssh/config/` or `/etc/ssh/ssh_config.d/SOME_CONFIG` is team-wide. in this case, using utilities like sshabu is not possible since it is not portable for those who use other tools. It would be very nice if sshabu could use the standard ssh client config.

Additionally, it would be nice if sshabu could itself check changes with the git remote repo and offer to do a pull/push/merge/rebase/etc to get them.

**Comment by alvtsky** [comment]
@13werwolf13  Clarification is needed.
Currently, sshabu converts sshabu.yaml to openssh.config and not vice versa. Are you interested in the possibility of overriding the target openssh.config (path/name) or having the ability to change the path of all configuration files?

To support collaborative work with sshabu.yaml, we were planning to introduce distributed key-value store support.

>  Additionally, it would be nice if sshabu could itself check changes with the git remote repo and offer to do a pull/push/merge/rebase/etc to get them.

As a quick and universal solution, I can suggest adding the functionality of a shell pre-run command. Does this meet your  requirements?

**Comment by 13werwolf13** [comment]
It's not enough for sshabu to convert its config to ssh config. If a change in the ssh config was made externally, it will be lost the next time it is changed via sshabu.

**Comment by alvtsky** [comment]
Yep, that's precisely why sshabu uses its own directory and openssh config. Thanks for your request. We will work on implementing these features.
